lib/starter/builder/templates/activerecord.rb

```diff
@@ -0,0 +1,117 @@
+# frozen_string_literal: true
+
+module Starter
+  module Templates
+    module ActiveRecord
+      def model_klass
+        'ActiveRecord::Base'
+      end
+
+      def initializer
+        <<-FILE.strip_heredoc
+        require 'yaml'
+        require 'erb'
```

Comment:
yeah, that was also my though, but no … ☹️

we can do the fine-tuning, after it is merged, ok?